tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -1010,11 +1064,22 @@ class CudnnRnnDescriptor : public CudnnDescriptorCommon<dnn::RnnDescriptor> {
     // Create the RNN handle
     cudnnStatus_t status = wrap::cudnnCreateRNNDescriptor(parent_, &rnn_desc_);
     CUDNN_RETURN_IF_FAIL(status, "Unable to create RNN descriptor");
+    #if CUDNN_VERSION >= 6000
+    // TODO: autotune algo selection.
```

Comment:
I don't think StreamExecutor is the right place to do auto-tuning; it's too low-level.  XLA and TF both implement auto-tuning atop SE for gemm, and I think we should do the same thing for convs.

Perhaps a better TODO for here is something like "allow the user to choose an algorithm".

Reply:
OK.